packages/medulas-react-components/src/components/Link/index.tsx

```diff
@@ -8,8 +8,14 @@ interface Props extends LinkProps {
 const Link = ({ children, to, ...rest }: Props): JSX.Element => {
   if (typeof to === 'string') {
     if (/^((mailto:)|((https?:)?\/\/))/.test(to)) {
+      // We use href and target in the <a> tag for styling and hover efects.
+      // The actual click event is handled via JavaScript to allow testing those clicks
+      const clickHandler = (event: React.MouseEvent): void => {
```

Comment:
This is interesting, I perfectly remember to be able to test <a> links in jest simulating the click properly, why do we need this?

Reply:
Because a click on an `<a href` does not emit a JavaScript `click` event, which can be mocked and tested. The only thing you can test is if the page changed to the desired url. But in this case, the URL is opened in a different window we have no access to.

I'd be very happy to revert and simplify if you show me a way how to test it. I searched the web for half an our or so and all the results test by checking `window.location.href`, which does not change when opening in a new window.

Reply:
Which is the faulty test? I need to understand the context and what you are trying to test.

Reply:
'redirects to the Terms and Conditions page'